apps/els_lsp/src/els_code_lens.erl

```diff
@@ -83,13 +93,10 @@ lenses(Id, Document) ->
 %% Constructors
 %%==============================================================================
 
--spec make_lens(atom(), els_dt_document:item(), poi()) -> lens().
-make_lens(CbModule, Document, #{range := Range} = POI) ->
-  Command = els_command:make_command( CbModule:title(POI)
-                                    , CbModule:command(POI)
-                                    , CbModule:command_args(Document, POI)),
+-spec make_lens(atom(), els_dt_document:item(), poi(), state()) -> lens().
+make_lens(CbModule, Document, #{range := Range} = POI, State) ->
   #{ range   => els_protocol:range(Range)
-   , command => Command
+   , command => CbModule:command(Document, POI, State)
```

Comment:
One callback instead of three. The price to pay is to invoke the `make_command` from the function, which I think makes sense. \o/